Content/Signing/signing.json

```diff
@@ -1,15 +0,0 @@
-{
-	"signing_withDeviceFactorSource_signTransaction": "Sign transaction with phone",
-	"signing_withDeviceFactorSource_idLabel": "Factor Source ID: %@",
-
-	"signing_useLedgerLabel_Ledger": "Ledger",
-	"signing_useLedgerLabel_lastUsed": "Last used",
-	"signing_useLedgerLabel_addedOn": "Added on",
-
-	"signing_signatureRequest_title": "Signature Request",
-	"signing_signatureRequest_body": "Make sure the following **Ledger hardware wallet device** is connected to a computer with a linked Radix Connector browser extension, and the Radix Babylon app is launched on the device.",
-	"signing_signatureRequest_instructions": "Review and sign the transaction on the Ledger device to continue.",
-
-	"signing_signatureSuccessful_title": "Signed",
-	"signing_signatureSuccessful_body": "Signature successful"
```

Comment:
we use these two in our codebase, but I want to ensure that we are actually showing them on the screen

Reply:
let me know, I don't think there are any screens that have these Strings.

Reply:
yes I can confirm wallet doesn't present these strings on the UI.